jsk_arc2017_baxter/euslisp/lib/util.l

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env roseus
+
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/util.l")
+
+(ros::load-ros-manifest "jsk_gui_msgs")
+(ros::load-ros-manifest "jsk_recognition_msgs")
+
+(defun opposite-arm (arm) (if (eq arm :larm) :rarm :larm))
```

Comment:
> what is it for?

To avoid name conflict and find the method easily, and consistency. We use `jsk_arc2017_baxter::baxter-robot` and `jsk_arc2017_baxter::apc-interface`.